utils/IrodsConnector.py

```diff
@@ -857,11 +867,11 @@ def irods_get(self, irods_path, local_path, **options):
 
         """
         if not self.icommands:
-            options = {
-                # FORCE_FLAG_KW: '',
+            opts = {
                 NUM_THREADS_KW: NUM_THREADS,
-                VERIFY_CHKSUM_KW: '',
+                VERIFY_CHKSUM_KW: ''
             }
+            options.update(opts)
```

Comment:
This was an incomplete implementation that I planned on returning to.  In the future I will annotate such code with:

```python
# TODO 
```